### PR TITLE
Sg 578 group tx help

### PR DESCRIPTION
**Pull request by aekong**
- Added two new paragraphs: Email addresses and Phone numbers, each containing paragraphs email address and phone number, respectively.
- Added twig components and related css for each added paragraph
- Exported config for new paragraphs
- tabs to spaces and typo cleanup

### CODE REVIEW

File: config/field.field.node.transaction.field_help.yml [hunk 1]
```diff
@@ -30,98 +36,109 @@ settings:
       help: help
       in_person_location: in_person_location
       phone: phone
+      email_addresses: email_addresses
+      phone_numbers: phone_numbers
```
Comment (jacine) [inline config/field.field.node.transaction.field_help.yml]: The ordering of this in the UI, is likely something that will be complained about. Not sure if it's worth messing with now, b/c I know the others will eventually be gone.

Screen Shot 2019-09-30 at 5 01 26 PM

File: web/themes/custom/sfgovpl/templates/paragraphs/paragraph--email--default.html.twig [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{#
```
Comment (jacine) [inline web/themes/custom/sfgovpl/templates/paragraphs/paragraph--email--default.html.twig]: There is no need to use `--default.html.twig`, `paragraph--email.html.twig` should be ok?

File: web/themes/custom/sfgovpl/templates/paragraphs/paragraph--phone--default.html.twig [hunk 3]
```diff
@@ -51,16 +51,14 @@ not paragraph.isPublished() ? 'paragraph--unpublished',
 %}
```
Comment (jacine) [inline web/themes/custom/sfgovpl/templates/paragraphs/paragraph--phone--default.html.twig]: There is no need to use `--default.html.twig`, `paragraph--phone.html.twig` should be ok.

File: web/themes/custom/sfgovpl/templates/paragraphs/paragraph--phone--default.html.twig [hunk 4]
```diff
@@ -51,16 +51,14 @@ not paragraph.isPublished() ? 'paragraph--unpublished',
...
-            } %}
-
-        {% endblock %}
-    </div>
+  <div {{ attributes.addClass(classes) }}>
+    {% block content %}
+      {% include "@theme/phone.twig" with {
+        "phone_owner": content.field_owner.0,
+        "phone_number": content.field_tel.0,
+        "phone_details": content.field_text.0
```
Comment (jacine) [inline web/themes/custom/sfgovpl/templates/paragraphs/paragraph--phone--default.html.twig]: I think it's better to let these render as opposed to drill to the zero, e.g. `content.field_whatever`.  For these, it's a little more work, but overriding the field templates, and just removing the wrappers is better.  You can do them individually or get all fields on x paragraph, like: `field--paragraph--phone-numbers.html.twig`

File: web/themes/custom/sfgovpl/templates/paragraphs/paragraph--email--default.html.twig [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+    not paragraph.isPublished() ? 'paragraph--unpublished'
+  ]
+%}
+{% block paragraph %}
+  <div{{ attributes.addClass(classes) }}>
+    {% block content %}
+      {% include "@theme/email.twig" with {
+          "email_title": content.field_title.0,
+          "email_address": content.field_email.0
```
Comment (jacine) [inline web/themes/custom/sfgovpl/templates/paragraphs/paragraph--email--default.html.twig]: I think it's better to let these render as opposed to drill to the zero, e.g. content.field_whatever. For these, it's a little more work, but overriding the field templates, and just removing the wrappers is better. You can do them individually or get all fields on x paragraph, like: `field--paragraph--email.html.twig`

File: config/field.field.node.transaction.field_help.yml (hunk 1 above)
Comment (aekong) [reply config/field.field.node.transaction.field_help.yml]: yup, will remove them in a future config export, after data's been migrated to the new grouped paragraph fields.  thx for the reminder! 😃

**Comment by aekong** [comment]
thanks for the tips!

**Pull request merged**